src/__tests__/__snapshots__/replacer-test.js.snap

```diff
@@ -278,7 +278,7 @@ exports[`replacer/replaceEmbedsInHtml replace audio embeds 1`] = `
           <button class=\\"c-dialog__close\\" type=\\"button\\">Lukk</button>
           <div class=\\"c-figure-license\\">
             <h3 id=\\"heading-audio-1\\" class=\\"c-figure-license__title\\">
-              Regler for bruk av lydklipp
+              Regler for bruk av lydfilen:
```

Comment:
Mulig dette er veldig pirk, men har kjørt opp denne lokalt, og viser denne artikkelen: http://localhost:3000/nn/article/22687

På lydavspilleren står det "Bruk av lydklipp" men inne i dialogen står det "Reglar for bruk av lydfila:" Det bør nok være samme ordet som brukes begge plassene. Under Regler for bruk for sjølve artikkelen brukes konsekvent lydfil så vi bør nok bruke det som standardord.

Reply:
Lager en pr på det i frontend-packages!